### PR TITLE
Fixes #1237. Add cityweb to spamlist

### LINKED ISSUE #1237: Block mailmanager.cityweb.de reports
It seems we are getting enough reports from mailmanager.cityweb.de to consider this spam now:
https://github.com/webcompat/web-bugs/issues?utf8=%E2%9C%93&q=is%3Aissue%20mailmanager.cityweb.de

We should update the [spam list](https://github.com/webcompat/webcompat.com/blob/master/webcompat/views.py#L168) to include this domain and change the message to be more generic.

### PR DESCRIPTION
**Pull request by adamopenweb**
Added cityweb to spam list and made the message more generic for multiple domains.

r? @miketaylr

### CODE REVIEW

**Comment by miketaylr** [comment]
This LGTM, but let's hold off on merging until I fix the tests on Master.

File: webcompat/views.py [hunk 1]
```diff
@@ -165,13 +165,14 @@ def create_issue():
...
-    spamlist = ['qiangpiaoruanjian']
+    # see https://github.com/webcompat/webcompat.com/issues/1237
+    spamlist = ['qiangpiaoruanjian', 'cityweb']
     for spam in spamlist:
         if spam in form.get('url'):
-            msg = (u'Anonymous reporting for qiangpiaoruanjian.cn '
-                   'is temporarily disabled. Please see '
-                   'https://github.com/webcompat/webcompat.com/issues/1141 '
-                   'for more details.')
+            msg = (u'Anonymous reporting for domain %s '
```
Comment (adamopenweb) [inline webcompat/views.py]: No problem, I'll change it.

**Comment by miketaylr** [comment]
This works as expected -- but should we put the whole domain, rather than just cityweb?

screen shot 2016-12-13 at 4 40 02 pm

**Comment by miketaylr** [comment]
@adamopenweb maybe use `cityweb.de` instead of just `cityweb`?

**Comment by miketaylr** [comment]
(oh wait, I can push that commit -- all good). LGTM.

**Pull request merged**